addons/pos_restaurant/static/src/overrides/components/receipt_screen/order_receipt/order_receipt.xml

```diff
@@ -2,9 +2,10 @@
 <templates id="template" xml:space="preserve">
 
     <t t-name="pos_restaurant.OrderReceipt" t-inherit="point_of_sale.OrderReceipt" t-inherit-mode="extension">
-        <xpath expr="//div[hasclass('pos-receipt-order-data')]" position="inside">
+        <xpath expr="//ReceiptHeader" position="after">
             <t t-if="props.data.isBill">
-                <div>PRO FORMA</div>
+                <div>Pro format receipt</div>
```

Comment:
```suggestion
                <div>Proforma Receipt</div>
```

I believe the term should be Proforma Receipt, but it would be great to get opinions from the PO before making this change!
cc - @mobt-odoo

Reply:
Hello @paso-odoo. 
The right term is "Pro forma" 

Source : https://grammarhow.com/proforma-pro-forma-or-pro-forma/#:~:text=The%20Cambridge%20Dictionary%20and%20The%20Oxford%20Dictionary%20list%20%E2%80%9Cpro%20forma%E2%80%9D%20as%20the%20only%20correct%20spelling.